experiments/rtp-hdrext/playout-delay/index.md

```diff
@@ -0,0 +1,56 @@
+
+
+MIN and MAX playout delay
+
+The playout delay on a frame represents the amount of delay added to a from the time it is captured at the sender to the time it is expected to be rendered at the receiver. Thus playout delay is essentially:
+
+Playout delay = ExpectedRenderTime(frame) - ExpectedCaptureTime(frame)
+
+MIN and MAX playout delay in turn represent the minimum and maximum delay that can be seen on a frame. This restriction range is best effort. The receiver is expected to try and meet the range as best as it can.
+
+A value of 0 for example is meaningless from the perspective of actually meeting the suggested delay, but it indicates to the receiver that the frame should be rendered as soon as possible. It is up-to the receiver to decide how to handle a frame when it arrives too late (i.e., whether to simply drop or hand over for rendering as soon as possible).
+
+RTP header extension format
+
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  ID   | len=2 |       MIN delay       |       MAX delay       |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+
+12 bits for Minimum and Maximum delay. This represents a range of 0 - 40950 milliseconds for minimum and maximum (with a granularity of 10 ms). A granularity of 10 ms is sufficient since we expect the following typical use cases:
+
+0: Certain gaming scenarios (likely without audio) where we will want to play the frame as soon as possible. Also, for remote desktop without audio where rendering a frame asap makes sense
+
+100/150/200: These could be the max target latency for interactive streaming use cases depending on the actual application (gaming, remoting with audio, interactive scenarios)
+
+400: Application that want to ensure a network glitch has very little chance of causing a freeze can start with a minimum delay target that is high enough to deal with network issues. Video streaming is one example.
+
+The header is attached on the RTP packet by the RTP sender when it needs to change the min and max smoothing delay at the receiver. Once the sender confirms the receipt of at least one RTP packet which has the min and max details, it MAY stop providing details on all further RTP packets until another change warrants communicating the details to the receiver again.  This is done as follows:
+
+RTCP feedback to RTP sender includes the highest sequence number that was seen on the RTP receiver. The RTP sender can track the sequence number on the packet that first had the playout delay extension and then stop sending the extension once the received sequence number is greater than the sequence number on the first packet containing playout delay.
```

Comment:
"containing playout delay" -> "containing the current values for playout delay in this extension".